com.icons8.Lunacy.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.icons8.Lunacy</id>
+  <name>Lunacy</name>
+  <summary>Free design software that keeps your flow with AI tools and built-in graphics</summary>
+  <metadata_license>Proprietary</metadata_license>
+  <project_license>Proprietary</project_license>
+  <description>
+    <p>
+      Lunacy is a next-gen vector graphic software for UI, UX, and web design. It has everything you are used to in other similar apps and goes beyond. Created by designers for designers to focus on the workflow and minimize distractions. No more wasting time surfing the internet for graphics or switching apps to remove backgrounds, or thinking placeholder texts. Lunacy has it all and even more. Faster than Figma, smarter than Sketch!
+    </p>
```

Comment:
the description should mention this is not an official flatpak.

Reply:
Great point.